UI/src/view/point_asset/trafficLayer.js

```diff
@@ -0,0 +1,53 @@
+(function(root) {
+    root.TrafficLayer = function(params) {
```

Comment:
I think TrafficSignLayer is a better name

Reply:
done